modules/installation-azure-finalizing-encryption.adoc

```diff
@@ -41,6 +41,39 @@ $ az role assignment create --role "<privileged_role>" \// <1>
 <1> Specifies an Azure role that has read/write permissions to the disk encryption set. You can use the `Owner` role or a custom role with the necessary permissions.
 <2> Specifies the identity of the cluster resource group.
 +
+. Obtain the `id` of the disk encryption set you created prior to installation by running the following command:
++
+[source,terminal]
+----
+$ az disk-encryption-set show -n <disk_encryption_set_name> <1>
+    --resource-group <resource_group_name> <2>
```

Comment:
`\` is needed, extra ` `  on line below is style :) (space is to align under the first arg)
```suggestion
$ az disk-encryption-set show -n <disk_encryption_set_name> \ <1>
     --resource-group <resource_group_name> <2>
```